scripts/bundleSizeConfig.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
-  MIN_SIZE: 547,
-  MAX_SIZE: 585,
+  MIN_SIZE: 639,
+  MAX_SIZE: 680,
```

Comment:
Increasing the bundle sizes by ~95KB is a significant increase. However, this is less than a single image on many of our pages and therefore I don't think it should be considered blocking. I've opened the issue https://github.com/bbc/simorgh/issues/4203 to address this in the medium-term.

Reply:
I think we should investigate bundle splitting this prior to merging

Reply:
these bundle sizes are not really representative - shouldn't it be the production size we care about, after minification and gzip?

Reply:
While its true that its not entirely representative of the gzipped size, its still a big indicator that this change is too big to pull into simorgh

Reply:
Attempting to bundle split this logic locally on the branch `attempt-bundle-splitting-for-CPS-asset`  which is [based off this branch]( https://github.com/bbc/simorgh/compare/use-psammead-rich-text-transforms...attempt-bundle-splitting-for-CPS-asset?expand=1)